numbagg/moving.py

```diff
@@ -10,36 +10,37 @@ def ewm_window_validator(arr, window):
 
 
 @ndmoving([(float64[:], float64, float64[:])], window_validator=ewm_window_validator)
-def ewm_nanmean(a, com, out):
+def rolling_exp_nanmean(a, alpha, out):
```

Comment:
Another option would be to have `a` and `out` be the first two arguments, so we could add additional arguments on following (`ignore_na`, `min_periods` / `min_weight`, etc)

Reply:
Given that numba requires the `out` arg at the end, this wouldn't work in the core function. We could adjust it in the decorator, and we can decide that later - the API is to pass `a` & `alpha`

So as long as we're OK with that API then we could move ahead with this, I think